ote/resources/public/language/en.edn

```diff
@@ -664,11 +665,11 @@
           "license-url" "License/terms of use URL"
           "external-interface" "External interface"
           "accessibility-info-url" "Accessibility description (WWW)"
-          "guaranteed-accessibility-description" "Guaranteed accessibility description"
+          "guaranteed-accessibility-description" "Kerro tarvittaessa lisätietoja aina saatavilla olevista esteettömyyspalveluista"
           "guaranteed-info-service-accessibility" "Guaranteed info service accessibility"
           "guaranteed-transportable-aid" "Guaranteed transportable aid"
           "guaranteed-vehicle-accessibility" "Guaranteed vehicle accessibility"
-          "limited-accessibility-description" "Limited availability accessibility description"
+          "limited-accessibility-description" "Kerro tarvittaessa lisätietoja rajoitetusti saatavilla olevista esteettömyyspalveluista"
```

Comment:
This input header is so long, that is getting on the input field:
![image](https://user-images.githubusercontent.com/988779/34080601-9f049150-e349-11e7-9926-217e9389ca9a.png)
I think we should increase the initial rows amount of free text fields to be bigger than one row.

Reply:
I leave these ui issues to Jussi